packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -760,7 +760,55 @@ describe('cluster', () => {
     cluster.addRotationSingleUser();
 
     // THEN
-    expect(() => cluster.addRotationSingleUser()).toThrow(/A single user rotation was already added to this cluster/);
+    expect(() => cluster.addRotationSingleUser()).toThrow(/A single user rotation for master secret is already added to this cluster/);
+
+
```

Comment:
Let's get rid of these empty lines.

Reply:
Almost all tests have this double line space. Do you still want me to remove it?

Reply:
Yes.